ios/LayoutReanimation/REASnapshot.m

```diff
@@ -24,6 +24,26 @@ - (instancetype)init:(UIView *)view
   return self;
 }
 
+- (instancetype)init:(UIView *)view withParent:(UIView *)parent
```

Comment:
- this method doesn't set `_view = view` like `init:` does - is this intentional?
- would it be possible to rewrite this to call `self = [self init:view]` and then applying the differences between these methods, or the other way around, having `init:` call `self = [self init:view withParent:[view superview]]`? This would reduce a bit of the shared logic between these functions.

Reply:
Resolved in: https://github.com/software-mansion/react-native-reanimated/pull/4014/commits/e0f1b6a7325627667feeb895e7501f320bc797c5